src/test/bin/test_exr2aces.py

```diff
@@ -16,31 +16,31 @@
 # no args = usage message, error
 result = run ([exr2aces], stdout=PIPE, stderr=PIPE, universal_newlines=True)
 print(" ".join(result.args))
-assert(result.returncode != 0)
-assert(result.stderr.startswith ("Usage: "))
+assert(result.returncode != 0), "\n"+result.stderr
+assert(result.stderr.startswith ("Usage: ")), "\n"+result.stderr
```

Comment:
I'm not a very fluent Python speaker. What does this mean

    assert(condition), string

?? Does the string print if the assertion fails? What about if the assertion succeeds? Is this an idiomatic usage that Python experts would use?

Reply:
Yeah basically it's just passing that string to the Exception. Same as if you did
`Exception("This is my error message")`
https://docs.python.org/3.9/reference/simple_stmts.html#the-assert-statement
I'm using "the extended form" here

So the string only prints if the assertion fails.

I think an alternative would be 
```
try:
    assert(result.returncode != 0)
except AssertionError:
    print(result.stderr)
    raise
```
But that just felt like it took up too many lines for what it did.

Reply:
No, it's fine, it just wasn't a convention I was familiar with